python/edm4hep/datamodel.py

```diff
@@ -0,0 +1,43 @@
+import ROOT
+res = ROOT.gSystem.Load('libedm4hepDict.so')
+if res < 0:
+    raise RuntimeError('Failed to load libedm4hepDict.so')
```

Comment:
I am wondering whether all these `RuntimeError`s should be `ImportError`s instead. In the end `import edm4hep` is the thing that will fail if one of these is triggered.

Reply:
Well typically for the libraries it will be because they are not in `LD_LIBRARY_PATH` which is a runtime search so one could say it's a `RuntimeError` ... both are quite generic, I don't think `ImportError` tells you much, that will also be quite obveious when checking the trace. So I don't have a strong opinion on this.

Reply:
Yeah, maybe a `RuntimeError` is better, because that doesn't point you towards a python problem in this case.